BrAddressForm.vue

```diff
@@ -284,6 +323,32 @@ export default {
   methods: {
     isString(str) {
       return isString(str);
+    },
+    countryFilter(val, done, abort) {
+      // reset addressCountry and the countries filter
+      this.value.addressCountry = '';
+      this.filter.countries = '';
+
+      if(val.length < 1) {
+        done();
+        return;
+      }
+      done(() => {
+        this.filter.countries = val.toLowerCase();
+      });
+    },
+    regionFilter(val, done, abort) {
```

Comment:
both of these are good (did not know you can abort a filter), but do need to be passed to @filter in their respective selects.